### ISSUE TITLE
some clarifications on acf tools:

### ISSUE THREAD

**Issue by Nickyg001** [RostiMelk/ACF-Tools#31]
some clarifications on acf tools:

1. It has limitations compared to plug-ins like the famous Advanced Custom Fields: Theme Code Pro

2. Because if I create a simple text field ACF: Theme Code Pro gives me this result
`<? php the_field ('intro_text'); ?>`

and ACF Tools gives me:
`<? php echo esc_url (get_field ('intro_text')); ?>`

what is the difference? why does ACF Tools return me an echo?
that I know the exact form should be `<? php the_field ('intro_text'); ?>` or am I wrong?

basically the code is a little different between the two where is the difference?

**Comment by RostiMelk** [comment]
What kind of limitations are you reffering to?
If you want any support for any 3rd party fields I recommend developing it and submitting a PR.

For now I have very little time to set a side for this project, so all new functionality that gets and had been built is made to speed up my own development workflow.

About the escaping of fields; it is good practice to escape fields in php to secure the field output. That's why I do it.
